### ISSUE TITLE
Multiple Monitor Support in Layout Screen?

### ISSUE THREAD

**Issue by thaiceman** [symless/synergy-core#6257]
Server: Windows 10 
Client:  Windows 10, OSX 10.13.3,  *nix coming soon™

2.0.5 - Stable On all devices

Not a bug but a much needed enhancement for end users running multiple monitors, it would be nice to have the extra monitors represented on the layout screen so it would be possible to tie a specific machine to the top/bottom of the extra monitor(s).

**Comment by dwmc15873** [comment]
As a paying customer: Oh please $deity, yes.

**Comment by fuzzzerd** [comment]
That'd be a great addition to the product.

**Comment by jonathan0104** [comment]
Would be a great feature, just purchased the Pro version and it was the first issue I found after such a simple setup. Coming back to the main computer from the secondary is flawless but going to the secondary computer puts the cursor on the 3rd monitor.

**Comment by YellowAfterlife** [comment]
I've had issues with this as well - my physical screen layout is like this
![image](https://user-images.githubusercontent.com/731492/37574395-34dada1e-2b29-11e8-8179-2402160892bc.png)
(a laptop and two monitors sitting on a shelf above it)

Occasionally I connect screen 3 to a Mac.

Since the layout is a grid, this is a problem - if I put the Mac on the left, it's treated like this
![image](https://user-images.githubusercontent.com/731492/37574494-de930284-2b29-11e8-8846-8ef384e44fae.png)
(top left corner of screen 1 transitions to middle right side of Mac's screen)

If I put it on the top (and move screen 2 to be in line with screen 1 on Windows), it's treated like this
![image](https://user-images.githubusercontent.com/731492/37574608-c1b8b23e-2b2a-11e8-9a35-98152822f025.png)

Placing all of them horizontally works as expected,
![image](https://user-images.githubusercontent.com/731492/37574653-175c9bd8-2b2b-11e8-86d6-417155eb14a6.png)
but isn't too comfortable to use because that's 5760 pixels from the one side to another.
Keyboard shortcut is only partial help for this, given the asymmetry.

Ideally should have a screen group positioning UI like OS themselves do.

**Comment by ylluminate** [comment]
**This is truly a critical option**. I have 9 monitors across 3 workstations. Interestingly this encompasses a request I've made for better or fine tuned control of monitor placement. I've found that it's nigh impossible to get monitors to where they need to go positionally and I'm still having to use Teleport.app between two of my macOS workstations and then Synergy on my middle (bridge) macOS client (set to host) to a Windows client.

![layout](https://cl.ly/442r1s0G233p/Image%202018-03-27%20at%2012.18.18%20PM.png)

Having the ability to control all of this graphically could resolve a LOT of requests and issues that folks have been having.

**_PLEASE NOTE_**: _It is important to be able to assign order to displays so that you can do abnormal things (only abnormal for Synergy; this is perfectly normal for macOS, for example) such as making the cursor from one display appear be linked to the top of another display. In the example above I move the cursor down from Group 2::Display 1 and it goes to the top of Group 2::Display 2.  This is necessary since Group 1::Display 1 MUST be the primary display and as a user, you do NOT want to have the cursor constantly accidentally going past the Menubar on macOS (the white long rectangle on Group 2::Display 1 & Group 1::Display 1) - when using macOS you need to quickly be able to bump-stop / hit the wall at the top of the primary display in order to get access to Menubar items and applets._

I'm also finding that my cursor keeps getting locked in the middle of the screen, which is causing terrible problems with some of my monitoring apps that are critical for work and so I've had to disable Synergy for the time being on one of my workstations since having the cursor default to center position, it wrecks havoc on the app's output.  I've been told by support that this is a known issue and is being resolved presently.

**Comment by KayoticSully** [comment]
I am a paying customer as well and would love this feature.  If Synergy2 gets this enhancement it will become critical infrastructure for me.

**Comment by elasim** [comment]
@nbolton can we share roadmap progress?

**Comment by JDTAY** [comment]
Throwing in my weight against this feature request, Synergy wasn't useable for me due to my multiple monitor setup at home and work and have had a refund processed because of it. When resolved, I'll get back on Synergy and repurchase. I'll monitor this issue and hope it's picked up soon.

**Comment by LukeSkyw** [comment]
The v2.x label was removed, does it means this feature is being dropped? I believe it is a must for synergy.

**Comment by nbolton** [comment]
Actually it's Synergy 2 that was dropped. We're working on the next major version, and we do plan on implementing this feature. 

> Once the next major version is ready for public use (we anticipate around 2022/2023), it will be released to general sale. All Synergy 2 customers will be upgraded for free, all lifetime customers (anyone who purchased before November 2017) will also benefit from the free upgrade, and of course, anyone who purchased early access will get it. This free upgrade covers over 85% of our current customer base.
> 
> Synergy 2 is no longer in development, but we are still developing Synergy 1 and we will be adding only new features to the next major version. After we release the next major version, we intend on maintaining Synergy 1 for a further two years to provide critical fixes and security updates.

Read more: https://symless.com/blog/putting-ux-first

**Comment by nbolton** [comment]
We do still plan on implementing this in Synergy 3, so I'll leave this open.
